services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -120,10 +120,10 @@ def init_app(settings: ApplicationSettings | None = None) -> FastAPI:
             add_oec_to_message=True,
         ),
     )
-    if settings.API_SERVER_DEV_FEATURES_ENABLED:
-        from ._profiler_middleware import ApiServerProfilerMiddleware
+    if settings.API_SERVER_PROFILING_MIDDLEWARE_ENABLED:
+        from servicelib.fastapi.profiler_middleware import ProfilerMiddleware
 
-        app.add_middleware(ApiServerProfilerMiddleware)
+        app.add_middleware(ProfilerMiddleware, app_name="api-server")
```

Comment:
don't we have a constant for "api-server"? something like  `_meta.APP_NAME` ?
I would use it everywhere

Reply:
Yep, good point. I will adjust